Sources/Fuzzilli/Execution/Execution.swift

```diff
@@ -59,4 +65,8 @@ public protocol Execution {
 
     /// Execution time in microseconds
     var execTime: TimeInterval { get }
+
+    /// A serialized version of (some of) the programs variables after
```

Comment:
Maybe we should call this `executionHash` or so? I think calling it something with `hash` makes it more clear that this is a hash of a number of values that were produced during the program's execution.